### PR TITLE
#29 - fix typo on loading message - Appolin Semegni Fotso

### LINKED ISSUE #29: BUG/1: Fix typo on loading message
The message we display when loading the sentences has a typo. Fix it

### PR DESCRIPTION
**Pull request by AppolinFotso**
…mponent

## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

I added the **currentTarget** event property to the button in SuggestionSentence component because the underline tag which is a child in the button element was creating a typo.  Additionally, I added LoadingSuggestions component to show a spinning image until the suggestions are ready to be viewed.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #29

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have carefully reviewed my own code
- [x] I have commented my code
- [x] I have updated any documentation

### CODE REVIEW

File: client/src/pages/Home.js [hunk 1]
```diff
@@ -17,6 +18,8 @@ export function Home() {
 	]);
 	const [selectedSuggestion, setSelectedSuggestion] = useState("");
 	const [nextOriginalText, setNextOriginalText] = useState(1);
+		const [loading, setLoading] = useState(1);
```
Comment (sztupy) [inline client/src/pages/Home.js]: make sure indentation is done properly across the whole file

File: client/src/pages/LoadingSuggestions.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const LoadingSuggestions = () => {
+	return (
+		<div>
+			<img src="../favicon.ico" alt="logo" className="loading" />
```
Comment (sztupy) [inline client/src/pages/LoadingSuggestions.js]: prefer not to use the `favicon.ico` for any purpose other than the fallback icon for legacy purposes. Prefer finding a transparent png for this purpose

Comment (AppolinFotso) [reply client/src/pages/LoadingSuggestions.js]: Done.

File: client/src/pages/Home.js (hunk 1 above)
Comment (AppolinFotso) [reply client/src/pages/Home.js]: Done.

**Comment by sztupy** [comment]
Tested locally, looks to be working

**Pull request merged**